core/common/src/main/java/alluxio/exception/UfsBlockAccessTokenUnavailableException.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.exception;
+
+/**
+ * The exception thrown when it fails to grab a UFS access token for a UFS block.
+ */
+public class UfsBlockAccessTokenUnavailableException extends AlluxioException {
```

Comment:
Old client won't trigger the server to throw this exception since our server is backward compatible. The old code path in the server is not deleted.